Platforms/OSX/AppKit/NSAccessibilityElement.swift

```diff
@@ -157,7 +157,7 @@ class NSAccessibilityElement : Object, NSAccessibility {
   @available(OSX 10.10, *)
   func accessibilityServesAsTitleForUIElements() -> [AnyObject]?
   @available(OSX 10.10, *)
-  func setAccessibilityServesAsTitleForUIElements(accessibilityServesAsTitleForUIElements: [AnyObject]?)
+  func setAccessibilityServesAsTitleFor(uiElements accessibilityServesAsTitleForUIElements: [AnyObject]?)
```

Comment:
That's a better discussion to have on swift-evolution. Either it's a consistent rule or it isn't, but the translation should be consistently lowercasing the first word.